workflow/kfdrc-rnaseq-workflow.cwl

```diff
@@ -0,0 +1,341 @@
+cwlVersion: v1.0
+class: Workflow
+doc: |
+  # Kids First RNA-Seq Workflow
+
+  This is the Kids First RNA-Seq pipeline, which includes fusion and expression detection.
+
+  ![data service logo](https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png)
+
+  ## Introduction
+  This pipeline utilizes cutadapt to trim adapters from the raw reads, if necessary, and passes the reads to STAR for alignment.
+  The alignment output is used by RSEM for gene expression abundance estimation.
+  Additionally, Kallisto is used for quantification, but uses pseudoalignments to estimate the gene abundance from the raw data.
+  Fusion calling is performed using Arriba and STAR-Fusion detection tools on the STAR alignment outputs.
+  Metrics for the workflow are generated by RNA-SeQC.
+
+  Inputs from this can be used later on for further analysis in joint trio genotyping and subsequent refinement and deNovo variant analysis. If you would like to run this workflow using the cavatica public app, a basic primer on running public apps can be found [here](https://www.notion.so/d3b/Starting-From-Scratch-Running-Cavatica-af5ebb78c38a4f3190e32e67b4ce12bb).
```

Comment:
Same as readme, this first sentence should go.

Reply:
Good catch. Removed.